### PR TITLE
telemetry: fix the bug about unnecessary error when run tidb only

### PR DESCRIPTION
**Pull request by YinWeiling**
### What problem does this PR solve?

Problem Summary:

### What is changed and how it works?

What's Changed:

fix the bug about unnecessary error when run tidb only

How it Works:

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)
run tidb only and observe

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM
- Breaking backward compatibility

### Release note 

- fix the bug about unnecessary error when run tidb only

### CODE REVIEW

**Review by wjhuang2016 — APPROVED** [review]
LGTM， but too many unrelated commits.

File: telemetry/data_window.go [hunk 1]
```diff
@@ -110,10 +109,7 @@ func readSQLMetric(timepoint time.Time, SQLResult *sqlUsageData) error {
 	promQL := "sum(tidb_executor_statement_total{}) by (instance,type)"
 	result, err := querySQLMetric(ctx, timepoint, promQL)
 	if err != nil {
-		if err1, ok := err.(*promv1.Error); ok {
-			return errors.Errorf("query metric error, msg: %v, detail: %v", err1.Msg, err1.Detail)
-		}
-		return errors.Errorf("query metric error: %v", err.Error())
+		logutil.BgLogger().Info("querySQLMetric got error")
```
Comment (xiongjiwei) [inline telemetry/data_window.go:112]: why not return `nil` here directly

Comment (YinWeiling) [reply telemetry/data_window.go:112]: Thx for your suggestion, I will accept it!

**Review by kaixu120811 — APPROVED** [review]
LGTM

**Review by breeswish — APPROVED** [review]

**Pull request merged**